src/createActions.js

```diff
@@ -0,0 +1,11 @@
+import ownKeys from './ownKeys';
+import createAction from './createAction';
+import camelize from './camelize';
+
+// creates a map of camelized actionTypes to the corresponding actions
+export default function createActions(actionTypes, transform = camelize) {
+  return ownKeys(actionTypes)
+    .reduce((acc, type) => (
+      Object.assign(acc, { [transform(type)]: createAction(type) })
```

Comment:
Can you use the [object spread operator](http://redux.js.org/docs/recipes/UsingObjectSpreadOperator.html) here, as it seems to be the idiomatic way to do this in the Redux community?

In the [Babel REPL with our presets](https://babeljs.io/repl/#?evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-0&code=const%20o%20%3D%20%5B'a'%2C%20'b'%2C%20'c'%5D.reduce((memo%2C%20value)%20%3D%3E%20%7B%0A%20%20return%20%7B%20...memo%2C%20%5Bvalue%5D%3A%20value%20%2B%20'%20value'%20%7D%0A%7D%2C%20%7B%7D)%0A%0Aconsole.log(o))

```js
const o = ['a', 'b', 'c'].reduce((memo, value) => {
  return { ...memo, [value]: value + ' value' }
}, {})

console.log(o) // {"a":"a value","b":"b value","c":"c value"}
```